### PR TITLE
Update README.md with clearer instructions

### PR DESCRIPTION
**Pull request by juliamuiruri4**
This pull request primarily involves updates to the `README.md` file to improve clarity and provide more detailed instructions. 

## Purpose
Key changes:

* [`README.md`](diffhunk://#diff-b335630551682c19a781afebcf4d07bf978fb1f8ac04c6bf87428ed5106870f5L35-R35): The sample data description now specifies that the documents are in .pdf and markdown format.
* [`README.md`](diffhunk://#diff-b335630551682c19a781afebcf4d07bf978fb1f8ac04c6bf87428ed5106870f5R78-R82): The command to clone the repo is now presented in a more readable format.
* [`README.md`](diffhunk://#diff-b335630551682c19a781afebcf4d07bf978fb1f8ac04c6bf87428ed5106870f5L103-R104): The requirements for running the sample locally with Ollama now include sufficient disk space.
* [`README.md`](diffhunk://#diff-b335630551682c19a781afebcf4d07bf978fb1f8ac04c6bf87428ed5106870f5L119-R126): The instructions for starting the application and uploading the PDF documents have been made more explicit.
* [`README.md`](diffhunk://#diff-b335630551682c19a781afebcf4d07bf978fb1f8ac04c6bf87428ed5106870f5L161-R162): The Azure prerequisites section no longer includes a link for Azure for Students. - Considering Azure OpenAI isn't yet available in the Azure for Students free subscription, so I think having it under pre-requisites would set a wrong expectation to those using the subscription.

## Does this introduce a breaking change?

```
[ ] Yes
[x] No
```

## Pull Request Type

What kind of change does this Pull Request introduce?

```
[ ] Bugfix
[ ] Feature
[ ] Code style update (formatting, local variables)
[ ] Refactoring (no functional changes, no api changes)
[ ] Documentation content changes
[ ] Other... Please describe:
```

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -32,7 +32,7 @@ This sample shows how to build a serverless ChatGPT-like experience with Retriev
 
 Building AI applications can be complex and time-consuming, but using LangChain.js and Azure serverless technologies allows to greatly simplify the process. This application is a chatbot that uses a set of enterprise documents to generate responses to user queries.
 
-We provide sample data to make this sample ready to try, but feel free to replace it with your own. We use a fictitious company called _Contoso Real Estate_, and the experience allows its customers to ask support questions about the usage of its products. The sample data includes a set of documents that describes its terms of service, privacy policy and a support guide.
+We provide sample data to make this sample ready to try, but feel free to replace it with your own. We use a fictitious company called _Contoso Real Estate_, and the experience allows its customers to ask support questions about the usage of its products. The sample data includes a set of documents, in .pdf and markdown format, that describes its terms of service, privacy policy and a support guide.
```
Comment (sinedied) [inline README.md:35]: we only have PDFs here, markdown is not support in this sample

File: README.md [hunk 2]
```diff
@@ -158,8 +159,7 @@ Note that the documents are uploaded automatically when deploying the sample to
 
 #### Azure prerequisites
 
-- **Azure account**. If you're new to Azure, [get an Azure account for free](https://azure.microsoft.com/free) to get free Azure credits to get started. If you're a student, you can also get free credits with [Azure for Students](https://aka.ms/azureforstudents).
-
+- **Azure account**. If you're new to Azure, [get an Azure account for free](https://azure.microsoft.com/free) to get free Azure credits to get started.
```
Comment (sinedied) [inline README.md:162]: We'd like to keep the specific addition for Students, as they don't have to use a credit card with the Azure for Students offer

File: README.md [hunk 3]
```diff
@@ -100,7 +101,7 @@ There are multiple ways to run this sample: locally using Ollama or Azure OpenAI
 
 ### Run the sample locally with Ollama
 
-If you have a machine with enough resources, you can run this sample entirely locally without using any cloud resources. To do that, you first have to install [Ollama](https://ollama.com) and then run the following commands to download the models on your machine:
+If you have a machine with enough resources (sufficient disk space), you can run this sample entirely locally without using any cloud resources. To do that, you first have to install [Ollama](https://ollama.com) and then run the following commands to download the models on your machine:
```
Comment (sinedied) [inline README.md:104]: it's not disk space, but rather a supported CPU/GPU for inference

**Review by sinedied — CHANGES_REQUESTED** [review]
Thanks for the PR! There are few small parts to fix, but this is definitely helpful!

File: README.md (hunk 1 above)
Comment (juliamuiruri4) [reply README.md:35]: Sorry, reverted this

File: README.md (hunk 3 above)
Comment (juliamuiruri4) [reply README.md:104]: Got it. I've removed the specification on resource

**Review by sinedied — APPROVED** [review]

**Comment by sinedied** [comment]
Thanks @juliamuiruri4 !

**Pull request merged**